volume17.xml

```diff
@@ -0,0 +1,215 @@
+                <dateline>
+                    <date when="1910-12-31">Saturday. Dec. 31st.</date>
+                </dateline>
+                <p>We have had a cold week. <persName ref="#Davis_Theodore_M">Theo</persName> spent his usual 3 days in the <placeName>Valley</placeName> &#8211; nothing has yet been found. We have seen a good deal of Mr. <persName>Whitaker</persName> and his party. They are on the old <persName>Cleopatra</persName> &#8211; but he has done a great deal to it and put about a great many pretty things, and <pb n="77"/> It looks very attractive. <persName>Mr.</persName> and <persName>Mrs. Newberry</persName> are with him, and they are to excavate somewhere near <placeName>Akmim.</placeName></p>
+                <dateline>
+                    <date when="1911-01-07">Saturday Jan. 7. 1911</date>
+                </dateline>
+                <p>The <orgName>"Miriam"</orgName> with the <orgName>Masperos</orgName> arrived on the 3rd. <persName ref="#Davis_Theodore_M">Theo</persName> and I called on them the next morning. <persName>Madame Maspero</persName> has been very ill on the way up with a bronchial attack. They are to stay 3 weeks. Mrs. <persName>McCormick</persName> and <persName>Miss Scott</persName> dined here tonight.</p>
+                <dateline>
+                    <date when="1911-01-14">Saturday. Jan. 14.</date>
+                </dateline>
+                <p>Mr. and <persName>Mrs. Buckley</persName> to lunch on Sunday, a warm fine day. Mr. <persName>Carter</persName> to tea one day. <persName>Harry</persName> as usual here for 2 days. <persName ref="#Davis_Theodore_M">Theo</persName> as usual in the <placeName>Valley</placeName> 3 days. Many visitors.</p>
+                <dateline>
+                    <date when="1911-01-21">Saturday. Jan. 21.</date>
+                </dateline>
+                <p>The usual weekly story &#8211; visitors, visits, teas and dinners. <persName>Lady Gorst</persName> and <persName>Miss Gorst</persName> here for tea one afternoon.</p>
+                <dateline>
+                    <date when="1911-01-28">Saturday. Jan. 28th.</date>
+                </dateline>
+                <p>A rather cold stormy week, with a little rain. <persName>Mrs. Hunter</persName> of <placeName>London</placeName> dined on Monday &#8211; Mrs. And <persName>Miss Warner</persName> of <placeName>Boston</placeName> and <persName>Miss Webb,</persName> a <persName>Marquise Guadalmina</persName> and her son, <orgName>Spaniards,</orgName> here to tea on Wednesday. <persName ref="#Davis_Theodore_M">Theo</persName> and I at <placeName>Mohassibs</placeName> one morning &#8211; met <persName>Miss Cassatt,</persName> the artist who lives in <orgName>Paris.</orgName> Mr. and Mrs. <persName>Kelekian</persName> came with her afterwards for tea. Mr <persName>Nicol</persName> dined on Friday. Cold and stormy.</p>
```

Comment:
multiple pers and place refs needed and org ref for "Spaniards"

Reply:
Do we want to tag nationalities? @ketchley